target-app/Chat/Views/ChatsViewController.swift

```diff
@@ -0,0 +1,92 @@
+        Match(matchID: 1, topicIcon: "ic_topic_movie", lastMessage: "¿Alguna película para recomendar?", unreadMessages: 1, user: User(id: 1 ,firstName: "name1", lastName: "last1", fullName: "Karen Bauer", name: "name 1", username: "user 1", email: "email1", gender: "gender1", password: nil, passwordConfirmation: nil, avatar: User.Avatar(smallThumbUrl: "avatar_mock_2")))
+    ]
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        applyDefaultUIConfigs()
+        setupNavigationBar()
+        setupTableView()
+    }
+    
+    func setupNavigationBar() {
+        navigationController?.setNavigationBarHidden(false, animated: true)
+        
+        navigationItem.titleView = UILabel(style: .secondary(text: "Chat"))
+        
+        let barLeftButtonItem = UIBarButtonItem(
+            image: UIImage(named: "ic_home_profile"),
+            style: .plain,
+            target: self,
+            action: nil
+        )
+        navigationItem.leftBarButtonItem = barLeftButtonItem
+        navigationItem.leftBarButtonItem?.tintColor = .black
+        
+        let barRightButtonItem = UIBarButtonItem(
+            image: UIImage(named: "ic_chat_map"),
+            style: .plain,
+            target: self,
+            action: #selector(tapOnMapBarButton)
+        )
+        navigationItem.rightBarButtonItem = barRightButtonItem
+        navigationItem.rightBarButtonItem?.tintColor = .black
+    }
+    
+    func setupTableView() {
+        view.addSubview(tableView)
+        
+        NSLayoutConstraint.activate([
+            tableView.topAnchor.constraint(equalTo: view.topAnchor),
+            tableView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            tableView.leftAnchor.constraint(equalTo: view.leftAnchor),
+            tableView.rightAnchor.constraint(equalTo: view.rightAnchor)
+        ])
+                
+        tableView.dataSource = self
+    }
+    
+    // MARK: - Actions
+
+    @objc
+    func tapOnMapBarButton(_ sender: Any) {
+        AppNavigator.shared.navigate(to: HomeRoutes.home, with: .changeRoot)
+    }
+
+}
+
+extension ChatsViewController: UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        data.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        if let cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath) as? ConversationTableViewCell {
+            cell.updateData(match: data[indexPath.row])
+            return cell
+        }
+        fatalError("could not dequeueReusableCell")
```

Comment:
instead of returning a fatalError that will cause a crash, we can dequeue a cell and cast it to the specific type we need in a different statement.
```suggestion
        let cell = tableView.dequeueReusableCell(withIdentifier: ConversationTableViewCell.reuseIdentifier, for: indexPath)
        if let cell = cell as? ConversationTableViewCell {
            cell.updateData(match: data[indexPath.row])
        }
        return cell
```